apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties

```diff
@@ -195,6 +195,8 @@ error.retry.code.invalid=Authentication failed. The code you entered is invalid
 authenticate=Continue
 no.valid.client.in.tenant=A valid client with the given client_id cannot be found in the tenant domain.
 not.authorized.to.use.requested.grant.type=The authenticated client is not authorized to use the requested grant type.
+authentication.flow.timeout=Sorry! Your sign-in attempt was taking too long
+authentication.flow.timeout.description=Looks like the sign-in process ended because of the delay. Let’s start again. If this problem persists, please contact the administrator.
```

Comment:
`â€™` -> is this how the apostrophe should be denoted?

Reply:
Looks like github rendering issue. If you check the code segment in comment, it's shown correctly